app/gui/view/component-browser/component-group/src/wide.rs

```diff
@@ -248,13 +252,16 @@ impl<const COLUMNS: usize> component::Frp<Model<COLUMNS>> for Frp {
                 entries <- input.set_entries.map(move |p| ModelProvider::<COLUMNS>::wrap(p, col_id));
                 background_height <+ entries.map(f_!(model.background_height()));
                 eval entries((e) column.set_entries(e));
-                _eval <- all_with(&entries, &out.size, f!((_, size) column.resize_and_place(*size)));
+                _eval <- all_with3(&entries, &out.size, &padding,
+                    f!((_, size, padding) column.resize_and_place(*size, *padding))
+                );
             }
             frp::extend! { network
                 out.is_mouse_over <+ is_mouse_over;
             }
 
-            let params = entry::Params { colors: colors.clone_ref() };
+            let params =
+                entry::Params { colors: colors.clone_ref(), selection_layer: default() };
```

Comment:
multi-line exprs. Please apply to other places in the code.

Reply:
Done. There are a few places with multiline expressions in the code, but it's impossible to change it without making code *way* uglier.